http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcClientConfiguration.java

```diff
@@ -651,4 +722,99 @@ public String getTokenSignatureAlgorithm() {
         return tokenSignatureAlgorithm;
     }
 
+    public String getAuthenticationRequestFormat() {
+        return authenticationRequestFormat;
+    }
+
+    public void setAuthenticationRequestFormat(String requestObjectType ) {
+        this.authenticationRequestFormat = requestObjectType;
+    }
+
+    public String getRequestSignatureAlgorithm() {
+        return requestSignatureAlgorithm;
+    }
+
+    public void setRequestSignatureAlgorithm(String algorithm) {
+        this.requestSignatureAlgorithm = algorithm;
+    }
+
+    public String getRequestEncryptAlgorithm() {
+        return requestEncryptAlgorithm;
+    }
+
+    public void setRequestEncryptAlgorithm(String algorithm) {
+        this.requestEncryptAlgorithm = algorithm;
+    }
+
+    public String getRequestEncryptEncValue() {
+        return requestEncryptEncValue;
+    }
+
+    public void setRequestEncryptEncValue (String enc) {
+        this.requestEncryptEncValue = enc;
+    }
+
+    public String getRealmKey () {
+        return realmKey;
+    }
+
+    public void setRealmKey(String key) {
+        this.realmKey = key;
+    }
+
+    public String getClientKeyStoreFile () {
+        return clientKeyStoreFile;
+    }
+
+    public void setClientKeyStoreFile(String keyStoreFile) {
+        this.clientKeyStoreFile = keyStoreFile;
+    }
+
+    public String getClientKeyStorePassword () {
+        return clientKeyStorePass;
+    }
+
+    public void setClientKeyStorePassword(String pass) {
+        this.clientKeyStorePass = pass;
+    }
+
+    public String getClientKeyPassword () {
+        return clientKeyPass;
+    }
+
+    public void setClientKeyPassword(String pass) {
+        this.clientKeyPass = pass;
+    }
+
+    public String getClientKeystoreType() {
+        return clientKeystoreType;
+    }
+
+    public void setClientKeystoreType(String type) {
+        this.clientKeystoreType = type;
+    }
+
+    public String getClientKeyAlias() {
+        return clientKeyAlias;
+    }
+
+    public void setClientKeyAlias(String alias) {
+        this.clientKeyAlias = alias;
+    }
+
+    public JsonWebKeySet getrealmKeySet() {
```

Comment:
Note that method names should use camel case, i.e., style wise, getrealm should be getRealm.

However, we should rename this to not mention realm as described in comments above.

Reply:
moved to utility class and renamed.